scene/gui/control.cpp

```diff
@@ -3576,6 +3675,14 @@ void Control::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "size_flags_vertical", PROPERTY_HINT_FLAGS, "Fill:1,Expand:2,Shrink Center:4,Shrink End:8"), "set_v_size_flags", "get_v_size_flags");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "size_flags_stretch_ratio", PROPERTY_HINT_RANGE, "0,20,0.01,or_greater"), "set_stretch_ratio", "get_stretch_ratio");
 
+	ADD_GROUP("Render Transform", "render_");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "render_offset", PROPERTY_HINT_NONE), "set_render_offset", "get_render_offset");
```

Comment:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "render_offset", PROPERTY_HINT_NONE, "suffix:px), "set_render_offset", "get_render_offset");
```

Reply:
I had this first but when using relative-to-size units it's not pixels anymore. Is there a way to dynamically set the suffix based on another property?

Reply:
Yes, using `_validate_property`

Reply:
But otherwise the `PROPERTY_HINT_NONE` is unnecessary 🙂

Reply:
I implemented it but I couldn't figure out yet how to make it re-evaluate the properties when `render_offset_relative_to_size` or `render_transform_pivot_relative_to_size` change. `notify_property_list_changed()` was freezing for a second and resetting scroll in the inspector so that's suboptimal :sweat_smile:

Reply:
That's the one, there's something else wrong if it does that

Reply:
Okay, added it. Thanks! :raised_hands: